pages/index.js

```diff
@@ -238,11 +219,10 @@ export default function Home() {
       <div className={walletAddress ? 'authed-container' : 'container'}>
         <div className="header-container">
           <p className="header">🖼 GIF Portal</p>
-          <p className="sub-text">
-            {!walletAddress ? "Upload" : "View"} your GIF collection in the metaverse ✨
+          <div className="sub-text">
+            {walletAddress ? "View" : "Upload"} your GIF collection in the metaverse ✨
```

Comment:
walletAddress instead of !walletAddress, more easy to read, just switched the conditional return values